src/hepqpr/qallse/qallse_base.py

```diff
@@ -313,31 +308,33 @@ def to_qubo(self, return_stats=False) -> Union[TQubo, Tuple[TQubo, Tuple[int, in
                                     key = (str(t1), str(t2))
                                     if key not in Q and tuple(reversed(key)) not in Q:
                                         if t1.eta_slice == eta and t2.phi_slice == phi:
-                                            #qubo_list[eta][phi][key] = self._compute_conflict_strength(t1, t2)
                                             Q[key] = self._compute_conflict_strength(t1, t2)
 
-
-
                 n_excl_couplers = len(Q) - n_vars
                 # 2b: inclusion couplers (consecutive doublets with a good triplet)
                 for q in quadruplets:
                     if q.eta_slice == eta and q.phi_slice == phi:
                         key = (str(q.t1), str(q.t2))
                         Q[key] = q.strength
-                        #qubo_list[eta][phi][key] = q.strength
 
-                #qubo_list[eta][phi] = Q[eta][phi]
-                quboslice[eta][phi] = QuboSlice(Q, eta, phi)
```

Comment:
Seems to me you don't need to change anything from the original code except introducing the eta,phi loop and the if-condition.